Alcatraz/Installers/ATZPluginInstaller.m

```diff
@@ -132,22 +132,35 @@ - (void)buildPlugin:(ATZPlugin *)plugin completion:(void (^)(NSError *))completi
 
 - (NSString *)findXcodeprojPathForPlugin:(ATZPlugin *)plugin {
     NSString *clonedDirectory = [self pathForDownloadedPackage:plugin];
-    NSString *xcodeProjFilename = [plugin.name stringByAppendingString:XCODEPROJ];
+    NSString *xcodeProjFilename = [plugin.name stringByAppendingPathExtension:XCODEPROJ];
+    NSMutableArray *allXcodeProjFilenames = [NSMutableArray new];
 
     NSDirectoryEnumerator *enumerator = [[NSFileManager sharedManager] enumeratorAtPath:clonedDirectory];
-    NSString *directoryEntry;
-
-    while (directoryEntry = [enumerator nextObject])
-        if ([directoryEntry.pathComponents.lastObject isEqualToString:xcodeProjFilename])
+    for (NSString *directoryEntry in enumerator)
+    {
+        NSString *fileName = directoryEntry.pathComponents.lastObject;
+        if ([fileName isEqualToString:xcodeProjFilename]) {
             return [clonedDirectory stringByAppendingPathComponent:directoryEntry];
+        } else if ([fileName.pathExtension isEqualToString:XCODEPROJ]) {
+            [allXcodeProjFilenames addObject:directoryEntry];
+        }
+
+        if ([directoryEntry isEqualToString:@".git"]) {
```

Comment:
I understand the performance gain from skipping `.git`, but isn't it premature optimization? Did you encounter a real-world case where scanning the `.git` folder was a problem?